src/typeahead/typeahead-window.ts

```diff
@@ -103,17 +157,59 @@ export class NgbTypeaheadWindow implements OnInit {
     } else {
       this.activeIdx--;
     }
-    this._activeChanged();
+    this._activeChanged(-1);
   }
 
   select(item) { this.selectEvent.emit(item); }
 
   ngOnInit() {
     this.activeIdx = this.focusFirst ? 0 : -1;
-    this._activeChanged();
+    this._activeChanged(null);
+  }
+
+  private _activeChanged(direction) {
```

Comment:
- can't we use a proper type here for the direction instead of -1 / +1?
- also am I not mistaken that you're not using the actual direction value anywhere?